bigbluebutton-html5/imports/ui/components/poll/container.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { makeCall } from '/imports/ui/services/api';
+import { withTracker } from 'meteor/react-meteor-data';
+import Users from '/imports/api/users';
+import Auth from '/imports/ui/services/auth';
+import Presentations from '/imports/api/presentations';
+import PresentationAreaService from '/imports/ui/components/presentation/service';
+import Polls from '/imports/api/polls';
+import Poll from './component';
+
+const PollContainer = ({ ...props }) => <Poll {...props} />;
+
+export default withTracker(({ }) => {
+  Meteor.subscribe('current-poll', Auth.meetingID);
+
+  const currentPresentation = Presentations.findOne({
+    current: true,
+  });
+
+  const currentSlide = PresentationAreaService.getCurrentSlide(currentPresentation.podId);
+
+  const currentUser = Users.findOne({ userId: Auth.userID });
```

Comment:
You could probably move most of those to a `service.js` like most of the big containers do.